packages/app/src/components/SearchResultsCard/index.js

```diff
@@ -1,42 +1,106 @@
-import React, { useContext, useEffect, useState } from 'react';
-import { Link } from 'react-router-dom';
-import { SearchResultDataContext } from '../../utils/context/SearchResultsContext';
-import './style.css';
+import React, { useContext, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { SearchResultDataContext } from "../../utils/context/SearchResultsContext";
+import "./style.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 const SearchResultsCard = () => {
-  const [SearchResultData, setSearchResultData] = useContext(SearchResultDataContext);
+  const [SearchResultData, setSearchResultData] = useContext(
+    SearchResultDataContext
+  );
+
+  SearchResultData.sort((a, b) => {
```

Comment:
Running logic in the function body of a component like this is generally not ideal because this logic will run every time the component renders. I'd probably create a separate state variable called `sortedSearchResults` or something and use that to store the sorted data. That way you only need to recompute the sort if the search results change.